xml/cap_depl_eks.xml

```diff
@@ -370,151 +288,347 @@ Custom TCP Rule    TCP          4443            0.0.0.0/0       CAP WSS
+suse            https://kubernetes-charts.suse.com/</screen>
+
+  <para>
+   List your chart names, as you will need these for some operations:
+  </para>
+
+<screen>&prompt.user;helm search suse
+NAME                        	VERSION	DESCRIPTION
+suse/cf-opensuse            	&latestscfchart;  A Helm chart for SUSE Cloud Foundry
+suse/uaa-opensuse           	&latestuaachart;  A Helm chart for SUSE UAA
+suse/cf                     	&latestscfchart;  A Helm chart for SUSE Cloud Foundry
+suse/cf-usb-sidecar-mysql   	1.0.1  	A Helm chart for SUSE Universal Service Broker ...
+suse/cf-usb-sidecar-postgres	1.0.1  	A Helm chart for SUSE Universal Service Broker ...
+suse/console                	&lateststratoschart;   A Helm chart for deploying Stratos UI Console
+suse/metrics                	1.0.0  	A Helm chart for Stratos Metrics
+suse/nginx-ingress          	0.28.3 	An nginx Ingress controller that uses ConfigMap...
+suse/uaa                    	&latestuaachart;  A Helm chart for SUSE UAA
+</screen>
+ </sect1>
+
+ <sect1 xml:id="sec.cap.install-uaa-eks">
+  <title>Deploy UAA</title>
+
+  <para>
+   Use &helm; to deploy the UAA (User Account and Authentication) server. You
+   may create your own release <literal>--name</literal>:
+  </para>
+
+<screen>&prompt.user;helm install suse/uaa \
+--name <replaceable>susecf-uaa</replaceable> \
+--namespace uaa \
+--values scf-config-values.yaml</screen>
+
+  <para>
+   Wait until you have a successful UAA deployment before going to the next
+   steps, which you can monitor with the <command>watch</command> command:
+  </para>
+
+<screen>&prompt.user;watch -c 'kubectl get pods --namespace uaa'</screen>
+
+  <para>
+   When the status shows RUNNING for all of the UAA nodes, proceed to deploying
+   &suse; &cf;. Pressing <keycombo> <keycap function="control"/>
+   <keycap>C</keycap> </keycombo> stops the <command>watch</command> command.
+  </para>
+
+  <important>
+   <title>Some Pods Show Not Running</title>
+   <para>
+    Some UAA and SCF pods perform only deployment tasks, and it is normal for
+    them to show as unready after they have completed their tasks, as these
+    examples show:
+   </para>
+<screen>
+&prompt.user;kubectl get pods --namespace uaa
+secret-generation-1-z4nlz   0/1       Completed
+          
+&prompt.user;kubectl get pods --namespace scf
+secret-generation-1-m6k2h       0/1       Completed
+post-deployment-setup-1-hnpln   0/1       Completed</screen>
+  </important>
+ </sect1>
+ 
+ <sect1 xml:id="sec.cap.install-scf-eks">
+  <title>Deploy SCF</title>
+    <para>
+      Because of the way EKS runs health checks, &cap; requires an edit to 
+      one of the <literal>scf</literal> &helm; charts, and 
+      then after a successful <literal>scf</literal> deployment remove a 
+      listening port from the EKS load balancer's listeners list.  
+  </para>
+  <para>
+      On your remote admin machine, look in
+      <filename>~.helm/cache/archive/cf-2.15.2.tgz</filename>. Extract the
```

Comment:
Using `helm fetch --untar` might be more supported here?

Reply:
Could do, I figure the charts are already downloaded so why not use them.

Reply:
Could do. The charts are already downloaded, so why not use them.